.github/workflows/ci.yml

```diff
@@ -66,10 +64,12 @@ jobs:
       matrix:
         TEST:
           - {CONTAINER: "stretch", TOXENV: "py27"}
-          - {CONTAINER: "stretch", TOXENV: "py35"}
+          - {CONTAINER: "ubuntu-bionic", TOXENV: "py36"}
     name: "${{ matrix.TEST.TOXENV }} on ${{ matrix.TEST.CONTAINER }}"
     steps:
       - uses: actions/checkout@v2
+      - run: python3 -m pip install -U pip
+        if: ${{ matrix.TEST.CONTAINER == "ubuntu-bionic" }}
```

Comment:
This is invalid syntax in an `if`. You can remove the `${{ }}` wrapper.

Reply:
That explains a lot, thanks 😄 🙈